.github/CODEOWNERS

```diff
@@ -66,7 +66,10 @@
 /sdk/containerservice/                                               @samkreter @zqingqing1 @GaneshaThirumurthi
 
 # PRLabel: %Cosmos
-/sdk/cosmos/                                                         @kushagraThapar @simorenoh @simplynaveen20 @xinlian12 @moderakh 
+/sdk/cosmos/                                                         @kushagraThapar @simorenoh @simplynaveen20 @xinlian12 @moderakh
+
+# ServiceLabel: %Cosmos %Service Attention
+#/<NotInRepo>/                                                       @gahl-levy @simorenoh
```

Comment:
Should this be:

```
# ServiceLabel: %Cosmos %Service Attention
# PRLabel: %Cosmos
/sdk/cosmos/                                                         @kushagraThapar @simorenoh @simplynaveen20 @xinlian12 @moderakh 
```

Reply:
That would leave out `@gahl-levy` from the bot role and would include a bunch of other folks in the future who aren't part of that rule.

The question, really, is what is the intent of the request that you received for the bot rule?   If the goal was to use the existing PR list and add the additional member then yes - we should use the form that you're suggesting and adjust the bot role to match.